package.json

```
@@ -1,42 +1,46 @@
 {
   "name": "witnet-toolkit",
-  "version": "2.0.10",
-  "description": "Basic typescript library for building Witnet Radon artifacts. This packag also bundles the `w3gw` and `witnet-toolkit` command-line tools.",
+  "version": "2.1.0-beta.0",
+  "description": "JS/TS library and CLI tooling for Web3 devs willing to interact with the Witnet network without actually running a Witnet node.",
   "author": "Witnet Foundation",
   "license": "MIT",
   "private": false,
   "type": "commonjs",
```

Add support to ECMAScript modules
```suggestion
  "type": "module",
```